test/e2e/upgrade/service/service.go

```diff
@@ -96,7 +96,10 @@ func (t *UpgradeTest) Setup(f *framework.Framework) {
 	// Hit it once before considering ourselves ready
 	ginkgo.By("hitting pods through the service's LoadBalancer")
 	timeout := service.LoadBalancerLagTimeoutAWS
-	service.TestReachableHTTP(tcpIngressIP, svcPort, timeout)
+	// require five passing requests to continue (in case the SLB becomes available and then degrades)
+	for i := 0; i < 5; i++ {
+		service.TestReachableHTTP(tcpIngressIP, svcPort, timeout)
```

Comment:
are the requests for reachability made back to back? or is there/should there be a delay between retries?

Reply:
They are, but five successive should catch both cases.  If this doesn’t fix the issue a short delay may be necessary. Or actually begin testing in setup and then stop.